### PR TITLE
feat: create UI components text area

### LINKED ISSUE #16: Create UI components `text area`
## Description

- The task is to create a shared UI component for `text area`.

## Criteria

- [ ] create a `text area` component

## Notes

- https://ui.shadcn.com/docs/components/textarea
image

## Assignment 

|Role|Assignee |
|---|---|
|`Developer`|@KaiKoide |
|`Reviewer`|@yuki-92c |

### PR DESCRIPTION
**Pull request by KaiKoide**
## Overview

Add text area component in the components/ui folder based on the design on Figma.

## Changes

- Install button component from shadcn/ui (https://ui.shadcn.com/docs/components/textarea) 
- style the design

## Review points

- The styling matches the Figma design

## Screenshots or videos

The images of icons in Figma
image

text area components
image

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)
- [x] The branch name follows our branch naming rules
- [x] The PR title follows our PR title rules
- [x] My code follows our coding style
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [ ] Code readability and simplicity
- [ ] Follows best practices and coding standards
- [ ] Understandable and maintainable code

### CODE REVIEW

File: src/components/ui/textarea.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export interface TextareaProps
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ className, ...props }, ref) => {
+    return (
+      <textarea
+        className={cn(
+          'flex h-[283px] w-full rounded-xl border border-input border-slate-400 bg-white px-3 p-px pb-[18px] text-lg ring-offset-background placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
+          className,
```
Comment (yuki-92c) [inline src/components/ui/textarea.tsx:14]: image

could you check the margin of the text box?

File: src/components/ui/textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export interface TextareaProps
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ className, ...props }, ref) => {
+    return (
+      <textarea
+        className={cn(
+          'flex h-[283px] w-full rounded-xl border border-input border-slate-400 bg-white px-3 p-px pb-[18px] text-lg ring-offset-background placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
```
Comment (yuki-92c) [inline src/components/ui/textarea.tsx]: image

could you check the margin of the text box?

**Review by yuki-92c — COMMENTED** [review]
thank you for your work and sorry for my late review

File: src/components/ui/textarea.tsx (hunk 2 above)
Comment (KaiKoide) [reply src/components/ui/textarea.tsx]: Thank you for review! I add margin to `text area` component!

Comment (yuki-92c) [reply src/components/ui/textarea.tsx]: could you please check the figma and adjust margin?

**Comment by yuki-92c** [comment]
could you please check the figma and adjust margin?

**Comment by KaiKoide** [comment]
> could you please check the figma and adjust margin?

I adjust the design based on Figma!

**Review by yuki-92c — APPROVED** [review]
thanks! Looks good to me.

**Pull request merged**